### PR TITLE
Consultations: fields improvements and back button in questions

### PR DESCRIPTION
**Pull request by LauraJaimeC**
#### :tophat: What? Why?
- Fields improvements and back button in question

## Button to return to questions list
imagen

## Remove fields in question
imagen

### CODE REVIEW

File: app/forms/decidim/consultations/admin/question_form.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module Decidim
```
Comment (tramuntanal) [inline app/forms/decidim/consultations/admin/question_form.rb:3]: can you document that using a decorator with class eval did not work?
also, can you document what is this override changing regarding the original¿

**Review by tramuntanal — CHANGES_REQUESTED** [review]
LGTM @LauraJaimeC just document the form object and that's all

File: app/forms/decidim/consultations/admin/question_form.rb (hunk 1 above)
Comment (LauraJaimeC) [reply app/forms/decidim/consultations/admin/question_form.rb:3]: I do it with a decorator! YEEAH!

**Review by tramuntanal — APPROVED** [review]
Good finding decorating the form object @LauraJaimeC !

**Pull request merged**